### ISSUE TITLE
Theme parsing error: csd.css

### ISSUE THREAD

**Issue by R-a-l-f** [lah7/gtk3-mushrooms#107]
Hi,

I use openbox on Arch Linux. The used theme is Adwaita.

If I run atril the GTK theme for all applications randomly changes into an abomination, with "Theme parsing error: csd.css" and the like.

See also [How to disable gtk3 client side decoration?](https://bbs.archlinux.org/viewtopic.php?id=292408) .

Regards,
Ralf

**Comment by R-a-l-f** [comment]
False alarm, I have now installed gtk3 only and no problems with a new user account, but my old user account still has this problem. Some config must still be fishy.

**Comment by lah7** [comment]
There isn't a `csd.css` on my system, which only ever had `gtk3` or `gtk3-classic` installed. Might be worth trying to find that leftover if present anywhere:

    sudo find / -type f -name "csd.css"

Or tracing the file which references it on the affected user account:

    grep -r "csd.css" ~/

since a CSS file can `@import` another file.

**Comment by R-a-l-f** [comment]
Hi,

the issue is most likely solved.

At first I've done a lot of

  grep -i csd .*

and similar

  grep -i csd

commands, to no avail.

After hours of testing with a newly created user account and my old user account, by disabling this and that on the old user account and coping over dirs and files to the new user account, I could always cause the problem by starting atril on my old user account, no matter if I tried gtk3 alone, gtk3 with gtk3-nocsd-git or gtk3-classic. The problem only rarely occurred in the newly created user account.

I finally removed dbus-broker-units and dbus-broker and installed dbus-daemon-units.

This seems to have solved the problem, regardless of whether I use gtk3 alone, gtk3 with gtk3-nocsd-git or gtk3-classic.

By the way, I have no client side decoration with programs like evolution, sylpheed, roxterm and many others, no matter if I use gtk3 alone, gtk3 with gtk3-nocsd-git or gtk3-classic. Roughly speaking, it doesn't seem to make any difference, but I went back to the old setup and use gtk3 with gtk3-nocsd-git.

The culprit was probably a combination of my openbox customised with scripts and dbus-broker.

Regards,
Ralf
